test/api_client/api_client.js

```diff
@@ -38,18 +38,11 @@ describe('APIClient module', () => {
 	];
 	const defaultHeaders = {
 		'Content-Type': 'application/json',
-		nethash: mainnetHash,
-		os: 'lisk-js-api',
```

Comment:
This might be useful for analytics.

Reply:
Yes, I was thinking about analytics recently and this might be very useful.